Patches/Turret Collection/TurretCollection_CE_Patch_Turrets.xml

```diff
@@ -596,14 +596,14 @@
 					</targetParams>
 				</Properties>
 				<AmmoUser>
-					<magazineSize>200</magazineSize>
+					<magazineSize>1500</magazineSize>
 					<reloadTime>7.8</reloadTime>
 					<ammoSet>AmmoSet_20x102mmNATO</ammoSet>
```

Comment:
Ammo quantity is fine, but it's come to my attention that the 'gatling gun turret' is firing _way_ too big around. Given the size of the turret, it should be firing a 7.62 or 5.56. It's currently firing the rounds intended CWIS, which is simply too large given the size of the turret.

Reply:
I originally interpreted the Turret Collection "Gatling Gun" as the M61 Vulcan, but after a second look, I have a feeling it could instead be made into a 7.62x51mm M134 minigun.

If the latter holds true - I'll probably overhaul the stats to be near-identical to the M134 miniguns from DMGT - pending the final decision on the 3000-round ammo capacity, of course.

Reply:
I agree that it should be the M134.

Reply:
Fixed in c947974